templates/submission/status-testcases.html

```diff
@@ -95,6 +95,12 @@ <h3>{{ _('Execution Results') }}</h3>
         {% if submission.is_graded %}
             <br>
             {% if submission.result != "AB" %}
+                <b>{{ _('Resources:') }}</b>
+                {% if submission.result != "TLE" %}
+                    <span title="{{ submission.time }}s">{{ submission.time|floatformat(3) }}s,</span>
```

Comment:
I'd prefer this to say `---` rather than omitting it entirely, to match up the submission list's behaviour.